### ISSUE TITLE
Move cow supplies into Shop

### ISSUE THREAD

**Issue by jeremyckahn** [jeremyckahn/farmhand#191]
Once #189 is done, the cow supplies should be available for purchase in the Shop under the Supplies tab.

>I like this idea! As we discussed on Discord, we should also move the cow supplies into this tab (in a separate ticket, which I'll file now).

_Originally posted by @jeremyckahn in https://github.com/jeremyckahn/farmhand/issues/189#issuecomment-908337502_
